javascript/javascript_and_the_backend/using_rails_for_your_backend.md

```diff
@@ -24,7 +24,7 @@ Are you done? Good. Next, it's time to practice allowing your front end JavaScri
 
 1. [Check out "Using JavaScript in your Rails App" from Daniel Kehoe](http://railsapps.github.io/rails-javascript-include-external.html). It is long and covers a lot of ground, but it's got great content. Some of the stuff on dependencies can be skimmed, but pay attention to the `content_for` stuff at the bottom.
 
-2. [Refresh yourself on Rails AJAX from RailsGuides](http://edgeguides.rubyonrails.org/working_with_javascript_in_rails.html) (just skim the top few sections).
+2. [Refresh yourself on Rails AJAX from RailsGuides](https://guides.rubyonrails.org/v4.2/working_with_javascript_in_rails.html) (just skim the top few sections).
```

Comment:
If we can change this (and the other references) to the 6.1 docs per your suggestion, we can get this one merged :+1:

Reply:
Sorry busy week and I made some git mistakes I was trying to figure out!

Can I just clarify, should I sync this to the upstream again (which will create a commit for a merge request on my branch) in addition to the changes, or just simply update the links?

Reply:
@ayrt-n No worries at all, I appreciate you taking time to  make the course better :tada: Please feel free to ask any questions if you need, I'm happy to help.

In this case, it should be fine to simply push changes to your own branch on your fork and they'll automatically show up here. If there's conflicts, they'll show up and we can cross that bridge when we come to it (but it's unlikely). 

If you do want to sync with the upstream there would no harm in it, though I'm fairly confident no one eles has been working on this particular file in the meantime.

Reply:
Pushed up the changes, just let me know if there are any issue! @ChargrilledChook